perlmod/Fink/Validation.pm

```diff
@@ -901,13 +901,13 @@ sub validate_info_file {
 					print "Error: $pretty_field has Mac line endings. ($value)\n";
 					$looks_good = 0;
 				}
-				# Check for hardcoded /sw.
+				# Check for hardcoded /opt/sw.
 				open(INPUT, "<$value") or die "Couldn't read $pretty_field ($value): $!\n";
 				while (defined($patch_file_content=<INPUT>)) {
 					# only check lines being added (and skip diff header line)
 					next unless $patch_file_content =~ /^\+(?!\+\+ )/;
-					if ($patch_file_content =~ /\/sw([\s\/]|\Z)/) {
-						print "Warning: $pretty_field appears to contain a hardcoded /sw. ($value)\n";
+					if ($patch_file_content =~ /\/opt/sw([\s\/]|\Z)/) {
```

Comment:
Should the `/` between "opt" and "sw" be escaped-> `/\/opt\/sw`  (and also in line 1345 below)?

Reply:
I believe so.  Alternatively, we could switch the delimiter and use e.g. m|match-text| instead of /match-text/ .  I tend to like the latter approach when dealing with paths.

Reply:
Should the .info validation also check for /sw (assuming not everyone will jump to /opt/sw and might be developing on older platforms)? I think the existing regexp catches "/sw" in any string context (not just as a root-level path).

Reply:
I pushed escaping the `/`. But I just confirmed that the current code does catch `/sw` in any context, not just at the start of the string. I'll revert this part of the patch to just `/sw` later